### PR TITLE
enki.cpp: clean #ifdef

### PR DESCRIPTION
**Pull request by sebpouteau**
Delete duplication of groundTexture constructor call in python/enki.cpp.

### CODE REVIEW

File: python/enki.cpp [hunk 1]
```diff
@@ -189,10 +189,11 @@ static World::GroundTexture loadTexture(const std::string& fileName)
 	QImage gt(QGLWidget::convertToGLFormat(QImage(fileName.c_str())));
 	
 	#if QT_VERSION >= QT_VERSION_CHECK(4,7,0)
-	return World::GroundTexture(gt.width(), gt.height(), (const uint32_t*)gt.constBits());
+    const uint32_t* bits = (const uint32_t*)gt.constBits();
```
Comment (stephanemagnenat) [inline python/enki.cpp]: please indent with tabs

File: python/enki.cpp [hunk 2]
```diff
@@ -189,10 +189,11 @@ static World::GroundTexture loadTexture(const std::string& fileName)
...
 	#else
-	return World::GroundTexture(gt.width(), gt.height(), (uint32_t*)gt.bits());
+	unint32_t* bits = (uint32_t*)gt.bits();
 	#endif
+    return World::GroundTexture(gt.width(), gt.height(), bits) ;
```
Comment (stephanemagnenat) [inline python/enki.cpp]: please indent with tabs

File: python/enki.cpp [hunk 3]
```diff
@@ -189,10 +189,11 @@ static World::GroundTexture loadTexture(const std::string& fileName)
 	QImage gt(QGLWidget::convertToGLFormat(QImage(fileName.c_str())));
 	
 	#if QT_VERSION >= QT_VERSION_CHECK(4,7,0)
-	return World::GroundTexture(gt.width(), gt.height(), (const uint32_t*)gt.constBits());
+	const uint32_t* bits = (const uint32_t*)gt.constBits();
 	#else
-	return World::GroundTexture(gt.width(), gt.height(), (uint32_t*)gt.bits());
+	unint32_t* bits = (uint32_t*)gt.bits();
```
Comment (stephanemagnenat) [inline python/enki.cpp]: It seems there is a typo, I do not think the type you want is `unint32_t*` but rather `uint32_t*`.

Comment (sebpouteau) [inline python/enki.cpp]: Sorry, I don't see this typo.

**Comment by stephanemagnenat** [comment]
Could you please rebase the PR on master into a single commit? Then I'll merge. Thank you!

**Pull request closed without merging**